src/redux/store.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from "@reduxjs/toolkit";
+import booksSlice from "./books/booksSlice";
+
+const store = configureStore({
+  reducer: {
+    books: booksSlice,
+    categories: "",
```

Comment:
- [ ] Kindly import the categories reducer as well and set up its reducer here also.